float64 degree
float64 maxDegreesPerSecond